CHANGELOG.md

```diff
@@ -23,9 +23,11 @@
 * Fixed a bug with the Docker dev build [PR #1376](https://github.com/catalystneuro/neuroconv/pull/1376)
 
 ## Features
+* Added `InscopixImagingInterface` for converting Inscopix imaging data. [PR #1361](https://github.com/catalystneuro/neuroconv/issues/1361)
```

Comment:
The link is wrong, it should be linking to a PR not an issue. 
```suggestion
* Added `InscopixImagingInterface` for converting Inscopix imaging data. [PR #1361](https://github.com/catalystneuro/neuroconv/pull/1361)
```

same in segmentation.